From d3ca1720ca52f639b9688a23577dc5c7286e81ad Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.liu@freescale.com>
Date: Wed, 12 Sep 2012 09:43:12 +0800
Subject: [PATCH 2088/2463] ENGR00223797-6 MX6 SabreSD:Initialize late init
 for IPUv3 fb pdata

This patch initializes late init field to false for IPUv3 fb pdata,
so we don't support late init by default unless we change them to
true in other places specificly.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
(cherry picked from commit cc9baf8aa84f4dd0120aa85960d254646a3aa523)
---
 arch/arm/mach-mx6/board-mx6q_sabresd.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabresd.c b/arch/arm/mach-mx6/board-mx6q_sabresd.c
index 3aabc75..18e5d1a 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabresd.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabresd.c
@@ -1240,6 +1240,7 @@ static struct ipuv3_fb_platform_data sabresd_fb_data[] = {
 	.mode_str = "LDB-XGA",
 	.default_bpp = 16,
 	.int_clk = false,
+	.late_init = false,
 	}, {
 	.disp_dev = "ldb",
 	.interface_pix_fmt = IPU_PIX_FMT_RGB666,
@@ -1252,12 +1253,14 @@ static struct ipuv3_fb_platform_data sabresd_fb_data[] = {
 	.mode_str = "CLAA-WVGA",
 	.default_bpp = 16,
 	.int_clk = false,
+	.late_init = false,
 	}, {
 	.disp_dev = "ldb",
 	.interface_pix_fmt = IPU_PIX_FMT_RGB666,
 	.mode_str = "LDB-VGA",
 	.default_bpp = 16,
 	.int_clk = false,
+	.late_init = false,
 	},
 };
 
-- 
1.7.7.4

